rustfmt.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
#964 updated all the bash scripts to have a consistent shebang. This line can be removed if you want.